WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1831,6 +1831,7 @@ ZmMailMsgView.prototype._renderMessageBody1 = function(params, part) {
         // calendar part in ICS format
         else if (ct === ZmMimeTable.TEXT_CAL) {
             content = ZmMailMsg.getTextFromCalendarPart(bp);
+            content = AjxStringUtil.htmlEncode(content);
```

Comment:
shouldn't the htmlEncode function be added into ZmMailMsg.getTextFromCalendarPart function to return encoded string ? There could be calls to this function from multiple places  which would get solved with that.